docs/md/expressions.md

```diff
@@ -0,0 +1,429 @@
+---
+id: expressions
+title: Column Expressions
+---
+
+<script src="../../../../js/index.js"></script>
+<link rel="stylesheet" href="../../../../css/concepts/index.css">
+
+Add new columns to an existing data view by using expressions based on (but not
+identical to) ExprTK ([source code](https://github.com/ArashPartow/exprtk),
+[homepage](http://www.partow.net/programming/exprtk/)). This can be considered
+similar to formulas in Excel, utilizing existing column values or arbitrary
+scalar values defined within the expression, as well as conditional and
+functional statements.
+
+## Basics
+
+### Creating a Custom Column
+
+Open a data view's side panel and press the "New Column" button, which will open
+the expression editor.
+
+<img src="/img/expressions/new-column.png" alt="" />
+
+### Naming a Custom Column
+
+A comment on the first line of an expression will be used as its name.
+
+```html
+// My Column Name
+```
+
+### Editing or Deleting a Custom Column
+
+Once a Custom Column is added, the expression editor can be opened by pressing
+the hamburger button beside the column name. The column may be deleted by
+pressing the X button, but to do so, it must first be removed from the data view
+by unchecking the box beside it.
```

Comment:
I would rephrase this ala "... Only `expressions` which are currently unused may be deleted"?

I would also mention that `expressions` which are unused are not calculated (and therefore don't impact performance at all), and that the `Reset` button does not delete `expressions` unless the `Shift` key is held down when it is clicked.

Maybe this section should also be moved immediately after `### Creating a Custom Column`, as it pertains to the interface rather than the syntax?